operators/operator_sdk/osdk-complying-with-psa.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="osdk-complying-with-psa"]
+= Complying with pod security admission
+include::_attributes/common-attributes.adoc[]
+:context: osdk-complying-with-psa.adoc
+
+toc::[]
+
+_Pod security admission_ is an implementation of the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[Kubernetes pod security standards]. Pod security admission restricts the behavior of pods. Pods that do not comply with the pod security admission defined globally or at the namespace level are not admitted to the cluster and cannot run.
+
+If your Operator project does not require escalated permissions to run, you can ensure your workloads run in namespaces set to the `restricted` pod security level. If your Operator project requires escalated permissions to run, you must set the pod security admission levels and security context constraints (SCC) for the Operator's namespace.
```

Comment:
you don't set SCCs for a namespace.

You set the allowed PSA level for the namespace, you set the allowed SCCs for the pod's(workload's) service account.

Reply:
Addressed in dee0dc7